src/sign-in-xplat/MainPage.xaml.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http.Headers;
+using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Essentials;
+
+namespace XPlat
+{
+    public partial class MainPage : ContentPage
+    {
+        //Set the scope for API call to user.read
+        private string[] scopes = new string[] { "user.read" };
+        private const string ClientId = "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]";
+        private const string Tenant = "[Copy the Tenant Id from the Azure portal]";
+        private const string Authority = "https://login.microsoftonline.com/" + Tenant;
+
+        // The MSAL Public client app
+        private static IPublicClientApplication PublicClientApp;
+
+        private static string MSGraphURL = "https://graph.microsoft.com/v1.0/";
+        private static AuthenticationResult authResult;

```

Comment:
hopefully this is only one missing convention @mmacy 

done | addressed from [c0337](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/c03379ea3f644a6556306ae964c8a92b269becd2)

Reply:
There's also `scopes`, but according to the linked conventions doc, all of these should be `_pascalCase` or `s_pascalCase` because they're private fields or private static fields.

Also, we should also prefer `readonly` over `const`, where possible (noted in point 3 in https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md).

Reply:
thanks for detailing the conventions over here @mmacy 

bug fixing them take 2 | [29266](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/29266a01a94550f5268a1d075ebb15781700fd6c)

Reply:
Other than the PublicClientApplication, I'm not sure they all need to be static, but this is looking better - thanks, @ferantivero!